Dockerfile-extra

```diff
@@ -2,7 +2,36 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-# Use fsfe/reuse:latest as a base, and install additional packages on top
-FROM fsfe/reuse:latest
+# Like normal Dockerfile, but install additional packages on top
 
-RUN apk --no-cache add openssh-client
+# Create a base image that has dependencies installed.
+FROM alpine:3.11 AS base
```

Comment:
Ah, is this not switched to `python` base, as well?

Reply:
The idea here is that this is like the normal Dockerfile (based on alpine) just with a few extra packages installed. See #238 for background.